packages/runtime/container-runtime/src/localStorageFeatureGates.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Local storage key to turn GC on / off.
+export const runGCKey = "FluidRunGC";
```

Comment:
I'd move these two keys out of this file to more appropriate place (ideally they stay only in container runtime and data stores does not need to know about them)

Reply:
The `DataStores` class needs to know about the gc test mode key to validate that we don't delete data stores if we are not in this mode.

Reply:
@vladsud I added a method to ContainerRuntime that tells whether its in GC test mode and DataStores calls it. Movded these keys to ContainerRuntime. Do you think this is better?